client/app/dedicatedCloud/datacenter/drp/summary/delete/dedicatedCloud-datacenter-drp-summary-delete.controller.js

```diff
@@ -0,0 +1,25 @@
+export default class {
+  /* @ngInject */
+  constructor($state, $timeout, $translate, Alerter, dedicatedCloudDrp) {
+    this.$state = $state;
+    this.$timeout = $timeout;
+    this.$translate = $translate;
+    this.Alerter = Alerter;
+    this.dedicatedCloudDrp = dedicatedCloudDrp;
+  }
+
+  confirm() {
+    this.isDeleting = true;
+    return this.dedicatedCloudDrp.disableDrp(this.drpInformations)
+      .then(() => this.$state.go('app.dedicatedClouds', {}, { reload: true }).then(() => {
+        this.$timeout(() => {
+          this.Alerter.success(this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_delete_drp_success'), 'dedicatedCloud_alert');
+        }, 5000);
```

Comment:
Why `5000` ?

Reply:
Necessary delay to display message in time